core/domain/user_services_test.py

```diff
@@ -88,20 +88,21 @@ def test_get_username_for_system_user(self):
             user_services.get_username(feconf.MIGRATION_BOT_USER_ID))
 
     def test_get_usernames(self):
-        gae_user_ids = ['test1', feconf.SYSTEM_COMMITTER_ID, 'test2']
-        usernames = ['name1', feconf.SYSTEM_COMMITTER_ID, 'name2']
-        user_emails = [
-            'test1@email.com', feconf.SYSTEM_EMAIL_ADDRESS, 'test2@email.com']
+        gae_ids = ['test1', 'test2']
+        usernames = ['name1', 'name2']
+        user_emails = ['test1@email.com', 'test2@email.com']
 
         user_ids = []
-        for gae_uid, email, name in python_utils.ZIP(
-                gae_user_ids, user_emails, usernames):
-            if gae_uid is not None:
-                user_id = user_services.create_new_user(gae_uid, email).user_id
-                user_services.set_username(user_id, name)
-                user_ids.append(user_id)
-            else:
-                user_ids.append(feconf.SYSTEM_COMMITTER_ID)
+        for gae_id, email, name in python_utils.ZIP(
+                gae_ids, user_emails, usernames):
+            user_id = user_services.create_new_user(gae_id, email).user_id
+            user_services.set_username(user_id, name)
+            user_ids.append(user_id)
+
+        # Check that system admin has correct username.
+        user_ids.append(feconf.SYSTEM_COMMITTER_ID)
```

Comment:
This is a bit confusing. I suggest having a separate test for system-ID-related usernames.

Also -- is the system username supposed to be the same as the ID? That should perhaps be documented in a comment somewhere (in the main code) -- perhaps above the feconf constant.

Reply:
Done.